bank/persistence/account_operations_test.go

```diff
@@ -0,0 +1,110 @@
+package persistence_test
+
+import (
+	"github.com/iliyanmotovski/bankv1/bank/domain"
+	"github.com/iliyanmotovski/bankv1/bank/persistence"
+	"gopkg.in/mgo.v2"
+	"gopkg.in/mgo.v2/bson"
+	"testing"
+	"time"
+)
+
+func TestItWritesAndReadsAccountDataFromDB(t *testing.T) {
+	session, _ := mgo.DialWithTimeout("localhost:27016", time.Second*10)
```

Comment:
This test is not going to work on a PC without local instance of mongodb. You may let users override database name by using an environment variable. 

https://github.com/clouway/godb/blob/master/datastoretest/mongo.go#L27